src/app/component/ImageUpload/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { useRef, useState } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { ReactComponent as CameraIcon } from '../../../img/camera.svg';
+import ImagePreview from './ImagePreview';
+import Button from '../Button/Button';
+
+export function ImageUpload({ styles, getFile = () => null }) {
+  const fileUpload = useRef();
+  const [file, setFile] = useState('');
+  const [preview, setPreview] = useState(null);
+
+  function handleFileChange({ target }) {
+    setFile(target.files[0]);
```

Comment:
Should we do a sanity check before setting and getting the file i.e. target.files.length && setFile()

Reply:
Maybe we could set a default value for the target, which could simplify the sanity check?